test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -112,6 +112,8 @@ def setUp(self):
     def run_test(self, model, input, rtol=1e-3, atol=1e-7, do_constant_folding=True,
                  batch_size=2, use_gpu=True, dynamic_axes=None, test_with_inputs=None,
                  input_names=None, output_names=None, fixed_batch_size=False):
+        model_ = model.copy() if hasattr(model, 'copy') else model
```

Comment:
Why exactly is this needed? If it has to do with the current situation with not being able to profile the model twice, won't this be a problem when we hit the `else` branch where there is no copy?

Reply:
That's a good point, I'm updating the else branch to use deepcopy. It would only hit the else branch if the model is not scripted. In that case it would be traced, instead of profiled. 
Also model copying is required for tracing as well, to avoid model attributes being updated by the export run.